README.md

```diff
@@ -31,10 +31,31 @@ Additionally, you can also explore the blockchain with [Starknet Stack Explorer]
 
 ## Quick start
 
+To run this locally through Docker, do:
 
+```
+make run-local
+```
+
+This will deploy 4 consensus nodes, the watcher-prover, and the blockchain Explorer. Before finishing, a client that sends a bunch of transactions is executed. The consensus nodes implement the Starknet RPC (partially for now), so you can curl the endpoints appropriately. You can also access the Madara explorer at http://localhost:4000/ and check out the blockchain. 
+
+When finished, don't forget to stop the containers by doing `make stop`.
+
+### Send transactions
+
+A mentioned above, as part of `make run-local`, a client that sends random transactions to the sequencer will run for a short while in order to populate the blockchain. You can also run the client on demand (`make run-client`).
+
+### Flow
+
+- The client sends random invoke transactions (either a fibonacci or factorial execution) to the consensus nodes which execute them with Cairo Native
+- Consensus nodes vote on blocks and commit them accordingly. If no transactions are sent, empty blocks are created regularly
+- In parallel, the watcher-prover is querying the RPC endpoints and checking transactions on blocks
+- When the watcher-prover gets a new block/transaction, it proves the execution through the CairoVM and the LambdaWorks prover
+- Proofs get saved either on the file system or on S3 (by default, the filesystem)
+- On the explorer, you can browse blocks and see the transactions they include. Additionally, {add proof verification explanation}
```

Comment:
Explanation still missing

Reply:
Yeah, was waiting on updates from the team but it seems it might not make it to tomorrow so I will strip it out